.circleci/config.yml

```diff
@@ -229,36 +192,35 @@ jobs:
               echo ""
               echo "lint files changed running lint"
               # python file (CommonServerPython lint is running over python 3 and 2)
-              demisto-sdk lint -i ./Packs/Base/Scripts/CommonServerPython
+              python3 -m demisto_sdk lint -i ./Packs/Base/Scripts/CommonServerPython
               
               # ps file
-              demisto-sdk lint -i ./Packs/Base/Scripts/CommonServerPowerShell
+              python3 -m demisto_sdk lint -i ./Packs/Base/Scripts/CommonServerPowerShell
               
         - store_artifacts:
             path: /tmp/lint_env_installation.log
   create-content-artifacts:
       docker:
-        - image: circleci/python:3.8.3-buster-node
+        - image: << pipeline.parameters.global-docker-image >>
       steps:
         - checkout
         - attach_workspace:
             at: ~/project
+        - <<: *install_demisto_sdk
         - run:
             name: Test Create Content Artifacts
             when: always
             command: |
-              . .tox/py37/bin/activate
-
               cd content
-              demisto-sdk -v
+              python -m demisto_sdk -v
               mkdir ./tmp
 
-              CIRCLE_BRANCH="master" demisto-sdk create-content-artifacts -a ./tmp
+              CIRCLE_BRANCH="master" python -m demisto_sdk create-content-artifacts -a ./tmp
```

Comment:
`CIRCLE_BRANCH` != `CI_COMMIT_BRANCH`? What are the uses of the two different environment variables? Do we need both of them?

Reply:
`CIRCLE_BRANCH` is used only in `deploy.sh`
`CI_COMMIT_BRANCH` is used only in `execute_test_content.py`. We might want to change the CIRCLE_BRANCH to CI_COMMIT_BRANCH and remove the other.

But it is for another pr.